### ISSUE TITLE
How to remove top bar and buttons

### ISSUE THREAD

**Issue by yangwao** [platformio/platformio-atom-ide-terminal#580]
I'm not familiar with Atom Themes, but should I make PR to plugin or in theme to remove top bar with buttons to have borderless terminal? Cheers, Thanks
Your plugin is great! 😸 

![image](https://user-images.githubusercontent.com/5887929/43366539-c2fa5440-933f-11e8-9d97-ddf72d861c51.png)

**Comment by the-j0k3r** [comment]
I personally find the hide and maximize functions helpful at times, so I dont think simply removing the `btn-toolbar` is a worthwhile consideration even if your proposal was optional.

Then you'de need to add those controls to the status bar when the `.btn-toolbar` is hidden and that may be too much overcrowding when you have few terminal instances open and if you have other addons making use of that space too.

Having said that, you can disable that locally for your setup, either if it bothers you or if you prefer a more clean layout by adding some CSS to your atom style.less

```css
.platformio-ide-terminal.terminal-view {
    .btn-toolbar {
        display: none !important;
    } 
   /* make input block smaller
     .input-block {
        padding: 2px !important;
    }
    */
}
``` 
![hidebtn-toolbar](https://user-images.githubusercontent.com/31389848/43397745-c631d14c-93fd-11e8-9d63-1c1331f46efb.gif)

Like I said making this part of platformio-ide-terminal is not ideal and thers too many issues already that need attention rather then introducing changes that could potentially add to the existing issues.

Hope that helps,  

TL:DR

We could use some bug squashing PR's though that make platformio-ide-terminal more robust and less buggy or add features like optional scrollbars especially since [this project is not under active development from its author](#543)

So I hope the above CSS will help you in the meanwhile.

**Comment by yangwao** [comment]
Thanks.
I'm just using single terminal for project and shortcuts helps me a lot to work without distraction, but sometimes I'm missing few height px on smaller screens.

Anyway, this block did the thing.

```css
.platformio-ide-terminal.terminal-view {
    .btn-toolbar {
        display: none !important;
    }
    .input-block {
        padding: 0px !important;
    }

}
```

**Comment by the-j0k3r** [comment]
YW.

You dont actually need the px suffix when the value is 0 ;)

**Comment by njordhov** [comment]
My preference is to have slimmer buttons and place the toolbar on top of the pane divider, using less space. The following styling does the trick:

```css
.platformio-ide-terminal.terminal-view {
    .input-block {
        padding: 0px !important;
        height: 8px;
        top: -4px;
    }
    .btn-toolbar {
        position: absolute;
        padding: 0px !important;
        height: 8px !important;;
        width: 100%;
        .btn-group {
          top: -12px;
          transform: scale(1, 0.35);
          z-index: 10;
        }
    }
}
```

Here is a screenshot:

![image](https://user-images.githubusercontent.com/219448/59475485-b50a3500-8e00-11e9-82c2-245c574168e1.png)
